From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Tue, 2 Jan 2024 10:23:29 +0100
Subject: ubuntu-spinner-style

# Description: Ubuntu style for the spinner/bgrt theme

# Description: Ubuntu style for the spinner/bgrt theme
# use the Ubuntu font, display the password entry under the bios logo
# Upstream: not-needed
---
 themes/bgrt/bgrt.plymouth.desktop | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/themes/bgrt/bgrt.plymouth.desktop b/themes/bgrt/bgrt.plymouth.desktop
index bfa79e7..51d8a6c 100644
--- a/themes/bgrt/bgrt.plymouth.desktop
+++ b/themes/bgrt/bgrt.plymouth.desktop
@@ -4,11 +4,11 @@ Description=Jimmac's spinner theme using the ACPI BGRT graphics as background
 ModuleName=two-step
 
 [two-step]
-Font=Cantarell 12
-TitleFont=Cantarell Light 30
+Font=Ubuntu 12
+TitleFont=Ubuntu Light 30
 ImageDir=@PLYMOUTH_THEME_PATH@/spinner
 DialogHorizontalAlignment=.5
-DialogVerticalAlignment=.382
+DialogVerticalAlignment=.7
 TitleHorizontalAlignment=.5
 TitleVerticalAlignment=.382
 HorizontalAlignment=.5
@@ -21,7 +21,7 @@ BackgroundStartColor=0x000000
 BackgroundEndColor=0x000000
 ProgressBarBackgroundColor=0x606060
 ProgressBarForegroundColor=0xffffff
-DialogClearsFirmwareBackground=true
+DialogClearsFirmwareBackground=false
 MessageBelowAnimation=true
 
 [boot-up]
